usaspending_api/common/data_connectors/async_spending_by_award_count.py

```diff
@@ -0,0 +1,32 @@
+import asyncio
+import asyncpg
+
+from django.db.models import Count
+
+from usaspending_api.awards.v2.filters.matview_filters import matview_search_filter
+from usaspending_api.common.helpers.orm_helpers import generate_raw_quoted_query
+from usaspending_api.common.helpers.sql_helpers import get_database_dsn_string
+
+
+async def run(sql):
+    conn = await asyncpg.connect(dsn=get_database_dsn_string())
+    removeal_index = sql.find("GROUP BY")  # Django refuses to provide a viable option to exclude "GROUP BY ..."
+    sql_result = await conn.fetch(sql[:removeal_index])
+    await conn.close()
+    print(sql_result)
```

Comment:
Probably should remove this print statement

Reply:
probably